### ISSUE TITLE
Mismatched URN and text in tlg2018.tlg012.opp-grc1

### ISSUE THREAD

**Issue by jduff-chs** [OpenGreekAndLatin/First1KGreek#270]
According to the TLG (the Perseus Catalog lacks this text) 2018.012 is Eusebius's *In Cantica Canticorum interpratio*, whereas in the file we have listed as such appears to be the third book of his *De ecclesiastica theologia* (the Klostermann edition, part of Band 4 of the Eusebius Werke series), which should be filed as 2018.009. However, we only have this third and final book of that work - there is no tlg2018.tlg009 currently in the repository.

As far as I can tell, the rest of tlg2018 is properly sorted, and there are no other gaps like this, or portions of 009 hiding anywhere. What should we do with this third book?

**Comment by jduff-chs** [comment]
For comparison and contrast, see here 1) the opening text of the *In Cantica Canticorum*, 2) the first few lines of XML in our file, and 3) the opening text of Book 3 of the *De ecclesiastica theologia*

1)  **Εὐσεβίου τοῦ Παμφίλου εἰς τὸ ᾆσμα τῶν ᾀσμάτων ἐξήγησις.**
Ἆσμα ᾀσμάτων διὰ τοῦτο καλεῖται, ἐπειδὴ μετὰ τὰ μέν τ' ἄλλα ᾄσματα
τοῦτο, μετὰ δὲ τοῦτο οὐκ ἔτι ἄλλο προσδοκᾶν ἐστίν· ἄλλο δὲ λέγω, οὐ βιβλίον, ἀλλὰ
τὸ ἐκ τοῦ βιβλίου σημαινόμενον. Ὃ δὲ λέγει, τοιοῦτον ἐστί. Πᾶσα θεία γραφὴ περὶ
τῆς εἰς ἡμᾶς τοῦ Λόγου καθόδου, καὶ τῆς ἐνσάρκου αὐτοῦ παρουσίας προφητεύει. 


2)
```xml
<body><div type="edition" n="urn:cts:greekLit:tlg2018.tlg012.opp-grc1">
<div type="textpart" subtype="book" n="3">
<div type="textpart" subtype="chapter" n="1">
<div type="textpart" subtype="section" n="1">
<p>Ὁ μὲν οὖν μέγας εὐαγγελιστὴς Ἰωάννης τὴν τοῦ σωτῆρος ἡμῶν <note type="marginal">&lt;ὡς διαστρόφως ὁ Μάρκελλος τὰς θείας γραφὰς ἑρμηνεύειν ἐτόλμα.&gt;</note>
θεολογίαν τὸν ἀποδοθέντα τρόπον διαφόρως εὐηγγελίζετο· Μάρκελλος Μάρκελλος
<lb n="40"/> δ’ ὅπως ταύτην ἀρνησάμενος ἐπὶ τὴν Ἰουδαϊκὴν ἐξώκειλεν
<note type="footnote">1 Γ’ vor Ζ. 38 in V | 3—37 die Kapitel so in m, in V steht bei den ersten ἐτόλμα.&gt;
beiden kein Zahlzeichen, bei Ζ. G β΄ usw. | 13 τὴν str. V2 doch vgl. S. 105, 3R | 27
τὸ V2 τε V* &lt;S. 174,9 R | 37 hierauf in V: κεφάλαια τοῦ ιݲ λόγου (sic). | 38–44 R m</note>
```

3)
![screen shot 2016-06-17 at 2 56 14 pm](https://cloud.githubusercontent.com/assets/19731635/16161669/c763777c-349c-11e6-971a-101cd53d639d.png)

**Comment by sonofmun** [comment]
If you are sure that this is from tlg2018.tlg009, then go ahead and rename it, change the metadata in the header, put it in its own folder tlg009 under tlg2018, add a `__cts__.xml` file for it (just use one of the other work level files as a template), and mark it up correctly as only book 3 of that work. It is no problem with CTS to have just part of a work in a file if we don't have the rest of the work. Once we get the rest, we should integrate it in to this file. Perhaps that should be something that we add to the 1st1KGreek list that we need to do.

**Comment by jduff-chs** [comment]
Sorry, still in the process of resolving this one! About to make a commit and pull request.

**Comment by jduff-chs** [comment]
Resolved as of PR #275!
